extensions/wikia/ContentReview/controllers/ContentReviewSpecialController.class.php

```diff
@@ -84,4 +104,39 @@ private function prepareReviewData( $reviewsRaw ) {
 
 		return $reviews;
 	}
+
+	private function prepareArchivedReviewData( $reviewsRaw, $reviewed ) {
+		$reviews = [];
+
+		foreach ( $reviewsRaw as $review ) {
+			$title = GlobalTitle::newFromID( $review['page_id'], $review['wiki_id'] );
+
+			$review['url'] = $title->getFullURL();
+			$review['title'] = $title->getBaseText();
```

Comment:
Can we use getText to distinguish subpages like Wikia.js/jsSliderNew.js? that are separate js pages.